### ISSUE TITLE
Train and test stage error!

### ISSUE THREAD

**Issue by erwangccc** [Diananini/LD-ConGR-CVPR2022#6]
There is error occurring when running train and test scripts. 
```
/mnt/sdb/user_homes/wangsc/workspace/LD-ConGR-CVPR2022/resnext-mmtm/offline_test.py:90: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  outputs = F.softmax(outputs_l[-1])
/mnt/sdb/user_homes/wangsc/.conda/envs/ld-congr/lib/python3.8/site-packages/sklearn/metrics/_classification.py:1327: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 in labels with no predicted samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))
```

It looks like the model inference nothing for the test data.
My config:

```
python offline_test.py 
--root_path /mnt/sdb/user_homes/wangsc/workspace/LD-ConGR-CVPR2022/resnext-mmtm/ 
--video_path /mnt/sdb/user_homes/wangsc/dataset 
--annotation_path annotation_hciGesture/dataV3/hciall_but_None.json 
--result_path results/hci_clf_test 
--resume_path results/hci-models/v3/hcigesture_mmtnet_1.0x_RGB-D_32/hcigesture_mmtnet_1.0x_RGB-D_32_best.pth 
--dataset hcigesture 
--sample_duration 32 
--model mmtnet 
--model_depth 101 
--resnet_shortcut B 
--batch_size 64 
--n_classes 10 
--n_finetune_classes 10 
--n_threads 4 
--modality RGB-D 
--n_val_samples 1 
--test_subset test 
--sample_size 112 
--iscrop 
--log_postfix _datav3_rgbd_32_crop
```

**Comment by Diananini** [comment]
These two are warnings, not errors, and do not affect training and testing. The first warning can be eliminated by specifying the   dimension:
`outputs = F.softmax(outputs_l[-1], dim=1)`
The second warning occurs when true positives + false positives = 0.

The test results can be found in results/hci_clf_test/test_datav3_rgbd_32_crop.log
